profiles/schema.py

```diff
@@ -0,0 +1,67 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
```

Comment:
concepts of interest also had [only some of the fields available via the API](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/api.py#L112)